src/goInstallTools.ts

```diff
@@ -115,9 +115,20 @@ function installTools(goVersion: SemVersion, missing?: string[]) {
 
 	outputChannel.appendLine(''); // Blank line for spacing.
 
+	// http.proxy setting takes precedence over environment variables
+	let httpProxy = vscode.workspace.getConfiguration('http').get('proxy');
+	let env = process.env;
+	if (httpProxy) {
+		env = Object.assign({}, process.env, {
+			http_proxy: httpProxy,
```

Comment:
@ramya-rao-a It's interesting that running 'npm t` on current master branch also gets 6 failed tests. I wonder if there are any dependencies have had a breaking change?